source/_patterns/04-components/menu/menu.twig

```diff
@@ -1,25 +1,25 @@
 <ul class="menu menu--{{menu_name}}">
   {% for item in items %}
     <li class="menu__item{% if item.children %} has-subnav{% endif %}">
-      <a class="menu__link{% if item.children %} has-subnav{% endif %}{% if item.current %} is-active{% endif %}{% if item.class %} {{ item.class }}{% endif %}" href="{{ item.link }}">{{ item.title }}</a>
-      {% if item.children %}
-        <ul class="menu menu__subnav">
-          {% for child in item.children %}
-            <li class="menu__item{% if child.children %} has-subnav{% endif %}">
-              <a class="menu__link{% if child.children %} has-subnav{% endif %}{% if child.current %} is-active{% endif %}{% if child.class %} {{ child.class }}{% endif %}" href="{{ child.link }}">{{ child.title }}</a>
-              {% if child.children %}
-                <ul class="menu menu__subnav">
-                  {% for grandchild in child.children %}
-                    <li class="menu__item">
-                      <a class="menu__link{% if grandchild.current %} is-active{% endif %}{% if grandchild.class %} {{ grandchild.class }}{% endif %}" href="{{ grandchild.link }}">{{ grandchild.title }}</a>
-                    </li>
-                  {% endfor %}
-                </ul>
-              {% endif %}
-            </li>
-          {% endfor %}
-        </ul>
-      {% endif %}
```

Comment:
Can you set your editor to use 2 spaces instead of 4 spaces or tabs?  The Gesso code base across all the versions uses 2 spaces for indentation.

Reply:
I ended up needing to turn off auto-detection for tabs/spacing. It was forcing Tabs (4) and even when I switched to Spaces (2), it would switch back to tabs. Should be formatted correctly now.

Reply:
@coreylafferty developers should be using the `.editorconfig` to ensure code formatting. (https://github.com/forumone/gesso-wp/blob/3.x/.editorconfig) Perhaps we just need to make sure everyone has their IDE setup to use that correctly. If it needs to be tweaked for Twig files then we may need to do that as well.